### DISCUSSION TITLE
ServiceUnavailableError

### DISCUSSION THREAD

**Discussion by letran3691** [outline/outline#2954] [Self Hosting]
I've enabled bebug=error 
when i's see log output below 

(node:25228) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
(Use `node --trace-deprecation ...` to show where the warning was created)
/var/www/outline/build/server/index.js:105
    throw err;
    ^

Error: bind EADDRINUSE null:3001
    at listenOnPrimaryHandle (node:net:1401:18)
    at rr (node:internal/cluster/child:155:12)
    at Worker.<anonymous> (node:internal/cluster/child:109:7)
    at process.onInternalMessage (node:internal/cluster/utils:49:5)
    at process.emit (node:events:402:35)
    at process.emit (node:domain:475:12)
    at emit (node:internal/child_process:917:12)
    at processTicksAndRejections (node:internal/process/task_queues:84:21) {
  errno: -98,
  code: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 3001
}

someone help me resolve this issue
thanks!

**Comment by tommoor** [comment]
The error is right there  – `EADDRINUSE` it means you already have something bound to the same port, you can either find and shutdown that service or change the port that Outline listens on with the `PORT` env.

**Reply by letran3691** [reply]
I've tried changing the port to 3002 it seems "Error: bind EADDRINUSE" don't display again, but the error still. 
![image](https://user-images.githubusercontent.com/19284401/150244655-ee84f7a0-7824-4e67-8956-ccea4b2b3eb7.png)

and has a WARNING in /static/main.770b928a35eb0a45605e.js is 20 MB, and won't be precached. Configure maximumFileSizeToCacheInBytes to change this limit. I don't have changes where
